.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: cpp
+compiler: gcc
+
+script:
+- g++ hello.cpp -o travis_gcc_cpp14
```

Comment:
Could you add `-Wall -Wextra -pedantic` here and maybe even `-Werror` (not only warn but stop compilation on errors).

And maybe use a bash loop to loop over all cpp files and compile them into separate binaries, and then execute them